api/hearings/models/hearingActuals.model.ts

```diff
@@ -0,0 +1,83 @@
+import { HearingResult, HMCStatus } from './hearings.enum';
+
+export interface HearingActualsModel {
+  hearingActuals: HearingActuals;
+  hearingPlanned: HearingPlanned;
+  hmcStatus: HMCStatus;
+}
+
+export interface HearingPlanned {
+  plannedHearingType: number;
+  plannedHearingDays: PlannedHearing[];
+}
+
+export interface PlannedHearing {
+  plannedStartTime: string;
+  plannedEndTime: string;
+  parties: Party[];
+}
+
+export interface Party {
+  partyId: number;
+  partyRole: number;
+  individualDetails: IndividualDetail;
+  organisationDetails: OrganisationDetails;
+  partyChannelSubType: string;
+}
+
+export interface IndividualDetail {
+  title: string;
+  firstName: string;
+  lastName: string;
+}
+
+export interface OrganisationDetails {
+  name: string;
+  cftOrganisationID: string;
+}
+
+export interface HearingActuals {
+  hearingOutcome: HearingOutcome;
+  actualHearingDays: ActualHearingDay[];
+}
+
+export interface HearingOutcome {
+  hearingFinalFlag: boolean;
+  hearingResult: HearingResult;
+  hearingResultDate: string;
+  hearingResultReasonType: string;
+  hearingType: string;
+}
+
+export interface ActualHearingDay {
+  hearingDate: string;
+  hearingStartTime: string;
+  hearingEndTime: string;
+  pauseDateTimes: PauseDateTime[];
+  actualDayParties: ActualDayParty[];
+}
+
+export interface PauseDateTime {
+  pauseStartTime: string;
+  pauseEndTime: string;
+}
+
+export interface ActualIndividualDetail {
+  firstName: string;
+  lastName: string;
+}
+
+export interface ActualOrganisationDetails {
+  name: string;
+}
+
+export interface ActualDayParty {
+  actualPartyId: number;
+  didNotAttendFlag: boolean;
+  actualIndividualDetails: ActualIndividualDetail;
+  actualOrganisationDetails: ActualOrganisationDetails;
+  partyChannelSubType: string;
+  partyId: number;
+  partyRole: number;
+  representedParty: number;
+}
```

Comment:
It's okay to put all those models together, could we name it with suffix xxxModel to consist with naming convention for all these models, just like the HearingActualsModel you have done.